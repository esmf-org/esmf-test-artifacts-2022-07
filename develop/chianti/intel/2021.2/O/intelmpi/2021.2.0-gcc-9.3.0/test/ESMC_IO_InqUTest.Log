build time -- 2022-06-16 02:05:31.843564
20220616 021402.919 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220616 021402.919 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220616 021402.919 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220616 021402.919 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220616 021402.919 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220616 021402.919 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220616 021402.919 INFO             PET0 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220616 021402.919 INFO             PET0 ESMF library build date/time: "Jun 16 2022" "02:04:35"
20220616 021402.919 INFO             PET0 ESMF library build location : /home/mpotts/intel_2021.2_intelmpi_O_develop
20220616 021402.919 INFO             PET0 ESMF_COMM                   : intelmpi
20220616 021402.919 INFO             PET0 ESMF_MOAB                   : enabled
20220616 021402.919 INFO             PET0 ESMF_LAPACK                 : enabled
20220616 021402.919 INFO             PET0 ESMF_NETCDF                 : enabled
20220616 021402.919 INFO             PET0 ESMF_PNETCDF                : disabled
20220616 021402.919 INFO             PET0 ESMF_PIO                    : enabled
20220616 021402.919 INFO             PET0 ESMF_YAMLCPP                : enabled
20220616 021402.919 INFO             PET0 Beginning Test, file ESMC_IO_InqUTest.C, line 59
20220616 021402.919 INFO             PET0 Beginning Test, file ESMC_IO_InqUTest.C, line 59NUMBER_OF_PROCESSORS 1
20220616 021402.921 INFO             PET0 PASS ScripInq, ESMC_IO_InqUTest.C, line 92
20220616 021402.921 INFO             PET0 PASS GridspecInq, ESMC_IO_InqUTest.C, line 112
20220616 021402.921 INFO             PET0 Ending Test, file ESMC_IO_InqUTest.C, line 116
20220616 021402.921 INFO             PET0 Finalizing ESMF
