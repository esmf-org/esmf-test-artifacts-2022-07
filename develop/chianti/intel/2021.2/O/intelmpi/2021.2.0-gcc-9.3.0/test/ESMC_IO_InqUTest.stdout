build time -- 2022-06-21 02:06:11.152925
Beginning Test, file ESMC_IO_InqUTest.C, line 59
Beginning Test, file ESMC_IO_InqUTest.C, line 59NUMBER_OF_PROCESSORS 1
grid_rank=2
grid_dims=[128,64]
PASS ScripInq, ESMC_IO_InqUTest.C, line 92
ndims=1
grid_dims=[320,160]
PASS GridspecInq, ESMC_IO_InqUTest.C, line 112
Ending Test, file ESMC_IO_InqUTest.C, line 116
Ending Test, file ESMC_IO_InqUTest.C, line 116 PET 0 Test Elapsed Time 2.313 msec.
